### ISSUE TITLE
Rename workspaces for consistency.

### ISSUE THREAD

**Issue by grallewellyn** [MAAP-Project/ZenHub#904]
This still needs some discussion with @anilnatha, but Alex suggested in a meeting that it was confusing naming the kernel "Python 3 (ipykernel)" because it isn't clear that the packages are from the `vanilla` or `pangeo` conda environments
We could name the kernel `Default vanilla` or `Default pangeo`, but we aren't sure what to do about the R workspace since there is one kernel for python and one for R

**Comment by wildintellect** [comment]
@grallewellyn `R python` and `R`?

**Comment by grallewellyn** [comment]
We discussed this and we were hesitant to name a kernel `R Python` because as of right now it cannot execute R code 
Correct me if I am wrong @anilnatha

**Comment by wildintellect** [comment]
Yes that's confusing, @rtapella have thoughts on how to make this more clear?

**Comment by chuckwondo** [comment]
How about naming them `Default (R)` and `Default (Python)`?

**Comment by anilnatha** [comment]
The platform team has also discussed the related issue of inconsistent naming of the "basic stable" workspace, currently `vanilla`. The user sees `vanilla` in terminal windows, but this name isn't mentioned anywhere else other than the list of running workspaces in Che. This can also be confusing.  So the team thought that naming this environment simply `python` would make sense as the name then aligns with the underlying base language capability.

This then begs the question, what about derivatives of these? For example, Pangeo is a flavor of our Python stack. Building on what @chuckwondo posted, I've prepared [this google sheet](https://docs.google.com/spreadsheets/d/16DIRmgBjWIMc6kWY7W2jEwtsfdruQtg2ygb8oELLaW4/edit#gid=0) containing a first-attempt at a naming convention we may follow. 

This is not final by any means, just wanted to highlight all the things that are impacted by any naming convention we adopt as it also will require updates to our documentation.

CC: @wildintellect @chuckwondo @grallewellyn @sujen1412 @marjo-luc @gchang @rtapella

**Comment by anilnatha** [comment]
The platform team has decided to leave the kernel names as is, and not rename the kernels.

That said though, I'm currently testing the renaming of "vanilla" to be "python" in our workspace configurations.

@rtapella If we have any references to the term `vanilla` in our documentation, we'll need to update those to be `python` after I've merged my changes into develop and tested everything once more.

**Comment by anilnatha** [comment]
After Grace merged the changes needed for this ticket, she noticed that the information displayed in ADE wasn't updated, notably the workspace names/descriptions/icons in addition to the dev files.

It was our understanding that the CI/CD takes care of this automatically, but maybe there is an issue with that process that we'd like @bsatoriu to look it.

**Comment by bsatoriu** [comment]
@grallewellyn @anilnatha see my PR here that addresses the dev file updates: https://github.com/MAAP-Project/maap-workspaces/pull/121

**Comment by rtapella** [comment]
So is this all deployed in UAT?

**Comment by rtapella** [comment]
(I am thinking about whether I want to redo screenshots etc. from UAT or just wait til it's in OPS and then release the updated documentation slightly behind release)

**Comment by rtapella** [comment]
Modified Getting Started and the DPS Tutorial with new screenshots and references to `python`
